### PR TITLE
course view with checkbox

### PR DESCRIPTION
**Pull request by marcinbutora**

### CODE REVIEW

File: training-service/src/app/content/course/component/course-list/course-list.component.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { CourseService } from '../../service/course.service';
 @Component({
   selector: 'app-course-list',
   templateUrl: './course-list.component.html',
-  styleUrls: ['./course-list.component.scss']
+  styleUrls: ['./course-list.component.css']
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.ts]: Please use scss, do not change to css

File: training-service/src/app/content/course/component/course-list/course-list.component.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.container {
+  margin-top: 1rem;
+}
+label.check {
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.css]: Please use only classes as selectors.

File: training-service/src/app/content/course/component/course-list/course-list.component.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  visibility: hidden;
+  pointer-events: none
+}
+
+label.check span {
+  padding: 7px 14px;
+  border: 2px solid #1f37e4;
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.css]: Please use variables for colors.

File: training-service/src/app/content/course/component/course-list/course-list.component.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+label.check input {
+  position: absolute;
+  top: 0;
+  left: 0;
+  visibility: hidden;
+  pointer-events: none
+}
+
+label.check span {
+  padding: 7px 14px;
```
Comment (holistic-fox) [inline training-service/src/app/content/course/component/course-list/course-list.component.css]: Please use responsive units

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/content/course/component/course-list/course-list.component.ts (hunk 1 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.ts]: done

File: training-service/src/app/content/course/component/course-list/course-list.component.css (hunk 2 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.css]: done

File: training-service/src/app/content/course/component/course-list/course-list.component.css (hunk 3 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.css]: done

File: training-service/src/app/content/course/component/course-list/course-list.component.css (hunk 4 above)
Comment (marcinbutora) [reply training-service/src/app/content/course/component/course-list/course-list.component.css]: done

**Pull request merged**